common/changes/@rushstack/heft-config-file/chore-replace-node-structuredclone_2025-04-24-03-50.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft-config-file",
+      "comment": "Fix Node 16 compatibility by using non-built-in structuredClone",
```

Comment:
If we want to support Node 16 reliably, Rush needs to re-add CI validation for this environment.  As discussed with @D4N14L it is not necessary for the entire Rush Stack monorepo to build with Node 16 (which will encounter a `typescript-eslint` incompatibility).  Instead, we could add a YAML step to the GitHub Action which merely invokes some Rush tests using Node 16 after building using the latest Node.

@sherlockfeng will consider implementing this, after evaluating the importance of Node 16 for his case.